.github/workflows/Deploy Gaffer On Kind

```diff
@@ -0,0 +1,91 @@
+
+name: Deploy Gaffer On Kind
+
+# Controls when the action will run. 
+on:
+  workflow_dispatch:
+  # Triggers the workflow to run at at 0900 Monday and Thursday day each week 
+  schedule:
+  - cron: "0 9 * * 1,4"
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: kubernetes
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Create kind cluster 
+        run: kind create cluster 
+      - name: Deploy Gaffer on kind
+        run: sh ../.github/workflows/execute-bash.sh docs/kind-deployment.md
+      - name: Send failure message to ms teams
+        if: ${{ failure() }}
+        uses: fjogeleit/http-request-action@master
+        with:
+           url: https://prod-100.westeurope.logic.azure.com:443/workflows/e77148a40306467683f9f5abec3ac060/triggers/manual/paths/invoke?api-version=2016-06-01&sp=%2Ftriggers%2Fmanual%2Frun&sv=1.0&sig=maJl7kx5roUQGhZNvbuyKGb5PCrYjmjaan8lyKOiP1M
+           method: 'POST'
+           contentType: "application/json"
+           data: "{'message': 'Gaffer deployment on kind failed'}"  
+  Test: 
+    needs: Build
+    runs-on: ubuntu-latest 
+    defaults:
+      run:
+        working-directory: store-implementation/accumulo-store/src/test/resources/
+    steps:     
+      - name: Checkout tools repo
+        uses: actions/checkout@v2
+        with:
+          repository: gchq/Gaffer
+      # Runs script to to execute bash commands in kind-deployment.md 
+      - name: Update properties files #1
+        run: sed -i '16 a gaffer.store.properties.class=uk.gov.gchq.gaffer.accumulostore.AccumuloProperties' store.properties
+      - name: Update properties files #2 
+        run: sed -i 's/localhost/localhost:58630/g' store.properties
+      - name: Update properties files #3 
+        run: sed -i 's/=user/=root/g' store.properties
+      - name: Update properties files #4 
+        run: sed -i 's/standardInstance/instance/g' store.properties
+      - name: Update properties files #5
+        run: sed -i '16 a gaffer.store.properties.class=uk.gov.gchq.gaffer.accumulostore.AccumuloProperties' store2.properties
```

Comment:
If the store class isn't updated, I think the tests will only run against the MiniAccumuloStore rather than the instance running in Kubernetes